src/curated-corpus/components/DropDownFilter/DropDownFilter.tsx

```diff
@@ -25,7 +26,15 @@ interface ScheduleDayFilterProps {
   /**
    * Callback to set filters on the Schedule Page
    */
-  setFilters: React.Dispatch<React.SetStateAction<ScheduleDayFilterOptions>>;
+  setScheduleFilters?: React.Dispatch<
+    React.SetStateAction<ScheduleDayFilterOptions>
+  >;
+  /**
+   * Callback to set filters on the Prospecting Page
+   */
+  setProspectFilters?: React.Dispatch<
+    React.SetStateAction<ProspectFilerOptions>
+  >;
```

Comment:
I wonder if this could be simplified so we don't pass two similar, but not quite 100% compatible setter methods to the newly abstracted dropdown filter component. Ideally it should have one setter. A reasonably quick way to refactor this without having to rework much code would be to have a single `setFilters` function in the component props, but let it accept either `ProspectFilterOptions` or `ScheduleDayFilterOptions`.